README.md

```diff
@@ -296,16 +296,16 @@ True
 * But why did the `is` operator evaluated to `False`? Let's see with this snippet.
   ```py
   class WTF(object):
-    def __init__(self): print("I ")
-    def __del__(self): print("D ")
+    def __init__(self): print("I", end = " ")
+    def __del__(self): print("D", end = " ")
```

Comment:
Hi, `end = " "` is python 3 specific. That's why I omitted it initially 😅

Reply:
It is the output which we should change IMO to avoid breaking this on Python 2.x

Reply:
If that's the case, then the output can be
```
I
D
I
D
True
```